src/main/java/web/components/CartProduct.java

```diff
@@ -0,0 +1,25 @@
+package web.components;
+
+import com.zebrunner.carina.webdriver.decorator.ExtendedWebElement;
+import com.zebrunner.carina.webdriver.gui.AbstractUIObject;
+import org.openqa.selenium.SearchContext;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.support.FindBy;
+
+public class CartProduct extends AbstractUIObject {
+
+    @FindBy(xpath = "./td[img]")
+    private ExtendedWebElement productImage;
+
+    @FindBy(xpath = "./td[text()='%s']")
+    private ExtendedWebElement cartProductTitle;
+
+    public CartProduct(WebDriver driver, SearchContext searchContext) {
+        super(driver, searchContext);
+    }
+
+    public String getCartProductTitle(String productTitle) {
+        return cartProductTitle.format(productTitle).getText();
+    }
```

Comment:
I think this method can return boolean instead of title:
```suggestion
    public boolean isProductTitlePresent(String productTitle) {
        return cartProductTitle.format(productTitle).isElementPresent();
    }
```
You format element with title and then get the same title again from the element

Reply:
The webpage is a little funny. 
```java
    @FindBy(xpath = "./td[text()='%s']")
    private ExtendedWebElement cartProductTitle;
```
    
There are no unique attributes in the cartProductTitle td element so this method is essentially a locator helper to find the specific td element. Otherwise i could locate this using an index locator like `td[2]` but felt it was not as specific as using the element text.

![image](https://github.com/jinocencio-solvd/LabaCarinaTest/assets/130516218/0eab3819-8424-4e42-9574-fb387479a534)

Reply:
After the demo today, I think i understand the implementation and use cases for isElementPresent better. Thanks I've made the changes as suggested.